configs/defis/config.yaml

```diff
@@ -324,6 +328,10 @@ pages:
     discussions:
       display: false
       create: false
+    datasets:
```

Comment:
I get where this comes from, but having "datasets" in a "datasets" section... We could name it "resources", "files" or something else, but it'd break the mapping with the API.

Let's keep it the way it is in this PR, but let's consider grouping all tab-related config later. @abulte WDYT?